apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -682,7 +692,7 @@ export class RushConfiguration {
    * will create the file.
    */
   public get commonVersions(): CommonVersionsConfiguration {
```

Comment:
>CommonVersionsConfiguration [](start = 31, length = 27)

Make sure `rush check` works with variants, since otherwise the `ensureConsistentVersions` setting (from rush.json) will be broken in a variant installation.

Reply:
Fixed `rush check`.